### ISSUE TITLE
Translate the AI-Declaration-Form into English

### ISSUE THREAD

**Issue by doerrdan** [dhbw-typst/oderso-template-dev#11]
The current AI declaration form is written in German. To ensure consistency and accessibility for all contributors and users, the entire form should be translated to English.

The file is called: `ai-declaration-form_dhbw-ma.typ`

**Comment by habetuz** [comment]
Will be available in `v2.1.0`
